### PR TITLE
clear up some code smells

### PR DESCRIPTION
**Pull request by seanpearsonuk**
Most of these are obvious but I've added some remarks.

### CODE REVIEW

File: src/ansys/fluent/visualization/matplotlib/matplot_windows_manager.py [hunk 1]
```diff
@@ -374,7 +374,6 @@ def close_windows(
 
     def _open_window(self, window_id: str) -> Union[Plotter, _ProcessPlotterHandle]:
         window = self._post_windows.get(window_id)
-        plotter = None
```
Comment (seanpearsonuk) [inline src/ansys/fluent/visualization/matplotlib/matplot_windows_manager.py:377]: unused

File: src/ansys/fluent/visualization/post_data_extractor.py [hunk 2]
```diff
@@ -104,9 +110,6 @@ def _fetch_contour_data(self, obj, *args, **kwargs):
         # contour properties
         obj._pre_display()
         field = obj.field()
-        range_option = obj.range.option()
-        filled = obj.filled()
-        contour_lines = obj.contour_lines()
```
Comment (seanpearsonuk) [inline src/ansys/fluent/visualization/post_data_extractor.py:109]: unused

File: src/ansys/fluent/visualization/pyvista/pyvista_windows_manager.py [hunk 3]
```diff
@@ -182,8 +182,8 @@ def _display_vector(self, obj, plotter: Union[BackgroundPlotter, pv.Plotter]):
             if "vertices" not in mesh_data or "faces" not in mesh_data:
                 continue
             mesh_data["vertices"].shape = mesh_data["vertices"].size // 3, 3
-            mesh_data[obj.vectors_of()].shape = (
-                mesh_data[obj.vectors_of()].size // 3,
+            mesh_data[vectors_of].shape = (
+                mesh_data[vectors_of].size // 3,
```
Comment (seanpearsonuk) [inline src/ansys/fluent/visualization/pyvista/pyvista_windows_manager.py:186]: use the local variable name

File: src/ansys/fluent/visualization/pyvista/pyvista_windows_manager.py [hunk 4]
```diff
@@ -198,12 +198,12 @@ def _display_vector(self, obj, plotter: Union[BackgroundPlotter, pv.Plotter]):
...
                     faces=mesh_data["faces"],
                 )
-            mesh.cell_data["vectors"] = mesh_data[obj.vectors_of()]
+            mesh.cell_data["vectors"] = mesh_data[vectors_of]
             scalar_field = mesh_data[obj.field()]
-            velocity_magnitude = np.linalg.norm(mesh_data[obj.vectors_of()], axis=1)
+            velocity_magnitude = np.linalg.norm(mesh_data[vectors_of], axis=1)
             if obj.range.option() == "auto-range-off":
                 auto_range_off = obj.range.auto_range_off
-                range = [auto_range_off.minimum(), auto_range_off.maximum()]
```
Comment (seanpearsonuk) [inline src/ansys/fluent/visualization/pyvista/pyvista_windows_manager.py:206]: shadowed builtin

File: src/ansys/fluent/visualization/pyvista/pyvista_windows_manager.py [hunk 5]
```diff
@@ -262,10 +261,7 @@ def _display_pathlines(self, obj, plotter: Union[BackgroundPlotter, pv.Plotter])
                 lines=surface_data["lines"],
             )
 
-            if node_values:
```
Comment (seanpearsonuk) [inline src/ansys/fluent/visualization/pyvista/pyvista_windows_manager.py:265]: redundant

**Review by hpohekar — APPROVED** [review]

**Comment by prmukherj** [comment]
My current fluent build is broken. I am fixing it with the develop build. I'll confirm the running if the repo code once and the approve the PR @seanpearsonuk .

**Review by prmukherj — APPROVED** [review]

**Pull request merged**